torch/csrc/jit/codegen/cuda/parser.cpp

```diff
@@ -3059,7 +3059,19 @@ class IrParser {
   }
 
   bool registerScalar(const JitValue* val) {
-    if (val->type()->isSubtypeOf(static_cast<c10::TypePtr>(FloatType::get()))) {
+    if (val->type()->isSubtypeOf(
+            static_cast<c10::TypePtr>(ComplexType::get()))) {
+      // NOLINTNEXTLINE(cppcoreguidelines-init-variables)
+      CgValue cg_val;
```

Comment:
nitpick, we can do nullptr to avoid nolint.

Reply:
Agree with the nit `CgValue cg_val = nullptr;`

Reply:
Oops. I don't know why I missed this comment. I have updated my PR.